latex/project-report.tex

```diff
@@ -562,22 +558,23 @@ \section{Conclusion}
 SSH keys.
 
 Next, we did a survey of the state of the code then and now to see if
-the introduction of such code was due to incompetence on the debian
+the introduction of such code was due to carelessness on the debian
 developer's part, or an understandable error that no automated tools
 were in place to catch. We concluded that there was no easy way to
 tell if the code modification would adversely affect the security of
-the library, and that, furthermore, the debian developer was given the
-go-ahead.
+the library, and OpenSSL currently has solid tests in place to mitigate
+the risk.
 
-The current state of the code is much better however, having more
+The current state of the code is much better; however, having more
 thorough unit-testing coverage that can catch more accidental errors
-made by OpenSSL developers or package distributors. Furthermore, the
+made by OpenSSL developers or package distributors could further
+improve OpenSSL. Furthermore, the
 current implementation of OpenSSL's random number generator is based
 on a specification and is therefore much easier to reason about.
 
-We concluded that the security of the OpenSSL tool has increased
-substantially since the 2008 debian OpenSSL bug was caught, the
-heartbleed vulnerability notwithstanding.
+Overall, we concluded that the security of the OpenSSL tool has increased
```

Comment:
I think 'overall' is redundant. This is the final paragraph in the conclusion so 'overall' is implied.